src/styles/interest.scss

```diff
@@ -0,0 +1,110 @@
+.interest {
+  display: flex;
+  justify-content: space-between;
+  overflow: hidden;
+
+  &__fact {
+    width: 610px;
+    margin-right: 120px;
+    border-top: 1px solid rgba(52, 79, 16, 0.5);
+  }
+
+  &__fact-text {
+    width: 380px;
+    padding: 100px 0 100px 230px;
+
+    font-weight: bold;
+    color: #333;
+
+    transition: padding $transition-duration;
```

Comment:
https://codeburst.io/low-performance-css-transitions-and-some-alternatives-f84ff35bf07